src/Elmish.WPF/Elmish.WPF.fsproj

```diff
@@ -42,7 +42,7 @@
     <Compile Include="Utils.fs" />
     <Compile Include="WindowState.fs" />
     <Compile Include="ViewModelArgs.fs" />
-    <Compile Include="BindingInternal.fs" />
+    <Compile Include="BindingData.fs" />
```

Comment:
And change the top-level module to match?

Reply:
They are already.